.gcloudignore

```diff
@@ -0,0 +1,17 @@
+# This file specifies files that are *not* uploaded to Google Cloud
+# using gcloud. It follows the same syntax as .gitignore, with the addition of
+# "#!include" directives (which insert the entries of the given .gitignore-style
+# file at that point).
+#
+# For more information, run:
+#   $ gcloud topic gcloudignore
+#
+.gcloudignore
+# If you would like to upload your .git directory, .gitignore file or files
+# from your .gitignore file, remove the corresponding line
+# below:
```

Comment:
Should we include things like .env and .env-sample like @theGaryLarson suggested in the other repo?

Reply:
@Seiyaroo I'd recommend adding the names of new env variables to the .env.sample file. If one needs to be created in your branch. Create a file titled .env.sample and add the variable names. After the name, put a one-line sentence describing how to generate the value. Can be a URL or a brief description.
e.g.
`MONGODB_URI = "connection string to your local or cloud MongoDB database"`